Neos.Flow/Classes/Utility/Environment.php

```diff
@@ -109,7 +109,7 @@ public function getMaximumPathLength(): int
      */
     public function isRewriteEnabled(): bool
     {
-        return (bool)Bootstrap::getEnvironmentConfigurationSetting('FLOW_REWRITEURLS');
+        return (string)Bootstrap::getEnvironmentConfigurationSetting('FLOW_REWRITEURLS') !== '0';
```

Comment:
That should not work. Why casting the result to string when the function has a bool return type declaration?

Reply:
read it again :)

Reply:
gnah, the binding ... 🙄